src/fidesops/core/config.py

```diff
@@ -128,6 +137,7 @@ class FidesopsConfig(FidesSettings):
     security: FidesopsSecuritySettings
     execution: ExecutionSettings
     root_user: RootUserSettings
+    admin_ui: AdminUiSettings
```

Comment:
Can you add this to the `log_all_config_values` helper too? Sorry I didn't know a clean way to introspect the settings object to do this automatically

Reply:
Yes, I'll make an issue for it now and update the code.